tests/gnrc_tcp/tests/shared_func.py

```diff
@@ -97,3 +101,13 @@ def verify_pktbuf_empty(child):
     pktbuf_size = child.match.group(2)
 
     child.expect(r'~ unused: {} \(next: (\(nil\)|0), size: {}\) ~'.format(pktbuf_addr, pktbuf_size))
+
+
+def sudo_guard():
+    sudo_required = os.environ.get("BOARD", "") != "native"
+    if sudo_required:
+        if os.geteuid() != 0:
```

Comment:
```suggestion
    if sudo_required and (os.geteuid() != 0):
```

saves you an indentation ;-)